src/vs/workbench/browser/parts/compositePart.ts

```diff
@@ -434,7 +434,7 @@ export abstract class CompositePart<T extends Composite> extends Part {
 		$(parent).div({
```

Comment:
I like how this change is in compositePart and not every viewlet has to do something special.